srcJava/com/rti/perftest/ddsimpl/RTIDDSImpl.java

```diff
@@ -1023,11 +1013,9 @@ private void configureReaderQos(
             drQos.durability.direct_communication = _directCommunication;
         }
 
-        if (("LatencyQos".equals(qosProfile) ||
-             "NoAckLatencyQos".equals(qosProfile)) &&
-            !_directCommunication &&
-            (_durability == 2 ||
-             _durability == 3)){
+        if (("LatencyQos".equals(qosProfile))
+                && !_directCommunication
+                && (_durability == 2 || _durability == 3)) {
```

Comment:
why dont we use:
- TRANSIENT_DURABILITY_QOS instead of 2
- PERSISTENT_DURABILITY_QOS instead of 3

REF: https://community.rti.com/static/documentation/connext-dds/5.3.0/doc/api/connext_dds/api_java/classcom_1_1rti_1_1dds_1_1infrastructure_1_1DurabilityQosPolicyKind.html#a0b69538f57900f159c71dce3b47890c0

Reply:
It is way easier to compare to the numbers (_durability is an integer) than compare to a DurabilityQosPolicyKind object, it would imply creating a DurabilityQosPolicyKind object and then compare.

Reply:
I tried that, but I got `error: valueOf(int) is not public in DurabilityQosPolicyKind; cannot be accessed from outside package`. How can I do this @alexcamposruiz?

Reply:
Do this instead:

    if (_durability == DurabilityQosPolicyKind.TRANSIENT_LOCAL_DURABILITY_QOS.ordinal()) ...

Reply:
This is great, thanks! Done.